src/profanity.c

```diff
@@ -94,17 +98,15 @@ char *saved_status;
 
 static gboolean cont = TRUE;
 static gboolean force_quit = FALSE;
-static gboolean gtk_enabled = FALSE;
 
 void
-prof_run(char *log_level, char *account_name, gboolean use_gtk)
+prof_run(char *log_level, char *account_name)
 {
-    gtk_enabled = use_gtk;
     _init(log_level);
     plugins_on_start();
-    if (gtk_enabled) {
```

Comment:
I guess you mean that we should check that gtk initialised ok and set this flag?

Sounds good if you want to add it back.